system/ad_service_state_monitor/config/ad_service_state_monitor.param.yaml

```diff
@@ -45,13 +44,6 @@
           type: "autoware_auto_mapping_msgs/msg/HADMapBin"
           transient_local: True
 
-        /map/pointcloud_map:
```

Comment:
Mm, I think you can't delete the setting for `autoware.launch`.

For `planning_simulator.launch`, yes it could be removed, but I wonder why it makes no problem now. :thinking: 
I guess that no pointcloud map is given in scenario simulations and the ad_service_state won't transition correctly.

I'll check the behavior.

Reply:
@shmpwk I've confirmed the behavior with the following commands.

```bash
ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/Downloads/sample-map-planning vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
ros2 bag record /initialpose /planning/mission_planning/goal
# Send an initial pose and goal pose from RViz.
```

```bash
ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/Downloads/sample-map-planning vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit pointcloud_map_file:=dummy
ros2 bag play {rosbag_dir}
# Change the target frame to `base_link`, click `Zero` to reset the viewpoint, and confirm the route is set.
ros2 topic echo /autoware/state
```

The results are:
- Before this PR, the state is 1.
- With this PR, the state is 4.

Reply:
Thank you for analyzing the behavior.